frontend/src/components/Header.vue

```diff
@@ -111,11 +110,9 @@ a {
   color: #fff;
   text-decoration: none;
 }
-.sysBar {
+.sys-bar {
```

Comment:
Since sys-bar isn't meaningful anymore can we rename this? .header?

Reply:
it's updated. style in this component is not scoped, so I use "app-header" to make the class name more unique.